src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5117,6 +5117,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ExplicitReadOnlyAttr" xml:space="preserve">
     <value>Do not use 'System.Runtime.CompilerServices.IsReadOnlyAttribute'. This is reserved for compiler usage.</value>
   </data>
+  <data name="ERR_ExplicitIsByRefLikeAttr" xml:space="preserve">
```

Comment:
I think we have other similar errors - for "dynamic" and tuple names and did not fold with those. I will check if it would make this scenario better.

---
In reply to: [118793105](https://github.com/dotnet/roslyn/pull/19795#discussion_r118793105) [](ancestors = 118793105)